manage/config-dialog.js

```diff
@@ -307,6 +369,12 @@ function configDialog(style) {
         }
       } else if (va.type === 'checkbox') {
         va.input.checked = Number(value);
+      } else if (va.type === 'range') {
+        const span = $('.current-value', va.input.closest('.config-range'));
```

Comment:
There are various places that would update `.current-value` box for the range input. Move them to a function. And we can also update it on input event so the value is updated immediately:
```diff
diff --git a/manage/config-dialog.js b/manage/config-dialog.js
index 697e174..0615cb1 100644
--- a/manage/config-dialog.js
+++ b/manage/config-dialog.js
@@ -315,7 +315,7 @@ function configDialog(style) {
   function updateVarOnChange() {
     if (this.type === 'range') {
       this.va.value = Number(this.value);
-      $('.current-value', this.closest('.config-range')).textContent = this.va.value + (this.va.units || '');
+      updateRangeCurrentValue(this.va, this.va.value);
     } else if (this.type === 'number') {
       // clamp stored value
       this.va.value = clampValue(this.value, this.va);
@@ -347,7 +347,15 @@ function configDialog(style) {
     return value;
   }
 
+  function updateRangeCurrentValue(va, value) {
+    const span = $('.current-value', va.input.closest('.config-range'));
+    span.textContent = value + (va.units || '');
+  }
+
   function updateVarOnInput(event, debounced = false) {
+    if (event.target.type === 'range') {
+      updateRangeCurrentValue(event.target.va, event.target.value);
+    }
     if (debounced) {
       event.target.dispatchEvent(new Event('change', {bubbles: true}));
     } else {
@@ -370,11 +378,8 @@ function configDialog(style) {
       } else if (va.type === 'checkbox') {
         va.input.checked = Number(value);
       } else if (va.type === 'range') {
-        const span = $('.current-value', va.input.closest('.config-range'));
         va.input.value = value;
-        if (span) {
-          span.textContent = value + (va.units || '');
-        }
+        updateRangeCurrentValue(va, value);
       } else {
         va.input.value = value;
       }
```
![test](https://user-images.githubusercontent.com/1324510/45755323-af583800-bc50-11e8-8ac5-af824a845221.gif)

Reply:
Done.